GitHub-resources/DIME-GitHub-Guides/clone-location.md

```diff
@@ -0,0 +1,211 @@
+your computer belongs to a large organization you are working for.
+Your access to this folder can change with little notice
+and you might therefore lose access to your clone.
+An almost universal convention across IT departments in most organizations
+is that your user folder in the path `C:\Users\` (Windows) or `/Users/` (Mac)
+is a folder you will always have access to.
+That is another reasons why `C:\Users\username\github`
+or `/Users/username/github` are excellent locations for all your clone.
+
+## How do I move a clone on my computer?
+
+There are two methods we recommend to move a clone on your computer.
+Method 1 is easiest but can only be done in some special circumstances.
+Method 2 can always be used.
+
+#### Method 1: Re-clone
+
+##### M1: Important - read first
+
+**Important**: When using this method you will lose anything not committed to GitHub.com.
+If you are unsure if that is acceptable in your case, use method 2.
+Content that **will be lost** when using this method includes:
+* Edits not yet first committed and then also pushed to GitHub.com
+* Any files ignored in your `.gitignore` file
+* Empty folders or folders with only ignored files
+
+Example of cases this method is appropriate to move clones on your computer includes:
+
+* Immediately after the clone was first cloned before any work has been done
+* A clone of a repository you never make any edits to,
+such as an open source tool you use in your code without editing it first
+
+
+##### M1: How to move the clone
+
+In this method you simply delete the clone and clone the repository again.
+When deleting the clone it is important that you delete the full folder.
+This means that if your repository is called `lyrics-clone`
+then you should delete the folder called `lyrics-clone` and all its content.
+
+If you open GitHub Desktop and select this repository as your current repository,
+then you will see the screen shown in the image below.
+Then simply click the button "_Clone Again_".
+If you do not see this screen or
+if you are not able to select the repository as current repository,
+then just go to GitHub.com and clone it again.
+
+<img src="https://github.com/worldbank/dime-github-trainings/blob/move-clone/GitHub-resources/DIME-GitHub-Guides/img/clone-not-found.png" width="75%"><!--- Image is read from master branch or use full URL-->
+
+This method can also be used if you have some issues with a clone
+you know that you have no un-committed/pushed and no ignored files and
+and want to start over fresh.
+And in this case you can of course clone to the same location.
+
+#### Method 2: Move the existing clone
+
+If you want to move the clone and keep all content,
+including content not pushed to GitHub.com
+such as un-committed edits and ignored files,
+then you need to use this method.
+
+###### M2: How to move the clone
+
+First start by copying the clone folder to the new location.
+Remember that you need to copy the folder as well as its content.
```

Comment:
I don't quite understand this remark. Are you clarifying that you need to copy the folder and not only the files and subfolders of the folder? If that's the case, I think it's enough to say that you need to copy the folder, because you couldn't possibly move the folder without moving its content as well

Reply:
```suggestion
Remember that you need to copy the folder and not just its content.
```
Haha, that is true. What I meant was again to repeat that both folder and content needs to be copied. I made it sound as if the risk was that the folder would be copied without the content, while the risk is the other way around. Let me know if this sounds better!

Reply:
yes, this is better!